general/1_terminal/Nazarii-Lysenko/Lysenko_HW1.md

```diff
@@ -0,0 +1,40 @@
+## Task1
+
+![Task1_1](task1_1.png)
+![Task1_2](task1_2.png)
+
+## Task2
+
+Directory changed for script, but not for terminal. Shell script is completely another
+process and “cd” command was executed for it. To check how this works we can run
+this code:
+```
```

Comment:
```suggestion
```sh
```

This is a markdown trick to add syntax highlighting to code blocks.

We can add any language here, for example, check out this Haskell snippet:

```hs
primes = filterPrime [2..]
  where filterPrime (p:xs) =
          p : filterPrime [x | x <- xs, x `mod` p /= 0]
```

Syntax highlighting depends on who renders your Markdown, but GitHub is quite good at it, even for more obscure languages. It definitely renders shell scripts well.

This is not mandatory, but just a nice trick for you to check out.

Reply:
You can learn more [on this GitHub doc page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks).

Reply:
Thanks!